ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -539,10 +539,9 @@ void RSkComponentTextInput::requestForEditingMode(bool isFlushDisplay){
       cursor_.end = 0;
     }
     privateVarProtectorMutex.unlock();
-    if (!caretHidden_) {
-       drawAndSubmit(isFlushDisplay);
-    }
   }
+  drawAndSubmit(isFlushDisplay);
```

Comment:
we will be triggering re-render here right ? can this be avoided and update of display string during OSK launch be handled seperately when OSK launch is called.

Reply:
Addressed